common/views/components/ApiToolbar/ApiToolbar.tsx

```diff
@@ -60,6 +67,44 @@ async function createTzitzitLink(imageId: string): Promise<Prop> {
   };
 }
 
+async function createTzitzitImageLink(
+  imageId: string
+): Promise<Prop | undefined> {
+  const apiUrl = `https://api.wellcomecollection.org/catalogue/v2/images/${imageId}?include=source.contributors`;
+  const image: Image = await fetch(apiUrl).then(res => res.json());
+
+  return setTzitzitParams({
+    title: image.source.title,
+    sourceLink: window.location.toString(),
+    licence: image.locations[0].license.id.toUpperCase(),
+    contributors: image.source.contributors,
+  });
+}
+
+async function createTzitzitWorkLink(
+  workId: string
+): Promise<Prop | undefined> {
+  const apiUrl = `https://api.wellcomecollection.org/catalogue/v2/works/${workId}?include=items,contributors`;
+  const work: Work = await fetch(apiUrl).then(res => res.json());
+
+  // Look at digital item locations only
+  const digitalLocation = work.items
+    ?.map(item =>
+      item.locations.find(location => location.type === 'DigitalLocation')
+    )
+    .find(i => i);
+
+  return setTzitzitParams({
+    title: work.title,
+    sourceLink: window.location.toString(),
+    licence:
+      digitalLocation?.type === 'DigitalLocation'
+        ? digitalLocation.license.id.toUpperCase()
```

Comment:
You could also pass in the `License` type and commonise the `.id.toUpperCase()`, but it's very minor.

Reply:
Yep love it, on its way

Reply:
@alexwlchan it was pushed, if you're happy with the change, I'll merge 👍

Reply:
Ah yeah go for it

My usual convention is that if I’ve given the green tick, I’m happy for it to be merged without another review from me. Any changes it needs are small enough to not need checking.

(I’m on a train into the office so can’t review code properly)

Reply:
Noted for future 👍 I'll merge